text/3028-cargo-binary-dependencies.md

```diff
@@ -1,4 +1,4 @@
-- Feature Name: (`bindeps`)
+- Feature Name: `(`bindeps`)`
```

Comment:
There are several find'n'replace errors like this

Reply:
I have gone through all of them again and fixed the mistakes. Must have overlooked them last time. Thanks for pointing that out.